liquibase-standard/src/main/java/liquibase/integration/spring/SpringLiquibase.java

```diff
@@ -252,6 +259,15 @@ public void afterPropertiesSet() throws LiquibaseException {
             Map<String, Object> scopeVars = new HashMap<>();
             scopeVars.put(Scope.Attr.ui.name(), this.uiService.getUiServiceClass().getDeclaredConstructor().newInstance());
             scopeVars.put(Scope.Attr.integrationDetails.name(), new IntegrationDetails("spring"));
+            if (!analyticsEnabled) {
+                log.info("Disabling Liquibase analytics.");
+            }
+            scopeVars.put(AnalyticsArgs.ENABLED.getKey(), analyticsEnabled);
+
+            if (this.licenseKey != null) {
+                log.info("Using PRO licenseKey defined in Spring configuration files.");
```

Comment:
Similarly, this feels overly verbose to me.

If it's kept, I don't think it should refer to "Spring configuration files". The only thing that `SpringLiquibase` knows is that `setLicenseKey` has been called. That could have been done by user code, due to a property in a file, or due to an environment variable.

Reply:
This one I"m bumping to fine/debug and removing the extra info. Thanks